proto/user.proto

```diff
@@ -57,6 +57,7 @@ message GrantPrivilege {
     uint32 source_id = 4;
     uint32 all_tables_schema_id = 5;
     uint32 all_sources_schema_id = 6;
+    uint32 view_id = 8;
```

Comment:
We might need to support grant/revoke privileges for views (and all views in schema maybe) in user service. But that's non-related to this PR. Cc @HuaHuaY